Updating Author and Contributor Information

Bhikule19 opened the pull request:
### Description
<!-- Please describe what you have changed or added -->

### Screenshots
<!-- if applicable -->

### Types of changes
<!-- What types of changes does your code introduce?  -->
<!-- Bug fix (non-breaking change which fixes an issue) -->
<!-- New feature (non-breaking change which adds functionality) -->
<!-- Breaking change -->

### How has this been tested?
<!-- Please describe in detail how you tested your changes. -->

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/ -->
- [ ] My code has proper inline documentation <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/ -->
- [ ] I've included any necessary tests <!-- if applicable -->
- [ ] I've included developer documentation <!-- if applicable -->
- [ ] I've added proper labels to this pull request <!-- if applicable -->

Bhikule19 committed:
Updating Author and Contributor Information

Bhikule19 committed:
Updated Tag version number

Nikschavan commented on README.md:
```diff
@@ -1,10 +1,9 @@
 **Requires at least:** 4.6
 **Tested up to:** 6.5
-**Stable tag:** 1.1.9
+**Stable tag:** 1.1.10
```
Changing readme only should not require a version bump

Nikschavan committed:
Update dependencies

Nikschavan committed:
Update tested upto to latest version

Nikschavan committed:
Update PHP versions in the CI

Nikschavan committed:
fix: Update CI to use PHP 7.4 and WordPress 6.9

- Change PHPUnit workflow from PHP 8.2/WordPress trunk to PHP 7.4/WordPress 6.9
- Update roots/wordpress dependency from ^6.8 to ^6.9
- Update composer.lock with PHP 7.4 compatible dependencies

Nikschavan committed:
fix: Add void return type to setUp methods for PHPUnit compatibility

Add `: void` return type to setUp() methods in all test files to fix
compatibility with Yoast PHPUnit Polyfills.

Nikschavan committed:
fix: Update assertions for PHPUnit 9 string methods

- Replace assertContains/assertNotContains with assertStringContainsString
  and assertStringNotContainsString for string comparisons
- Add .phpunit.result.cache to .gitignore

Nikschavan committed:
chore: Exclude tests directory from PHPCS checks

Test files use PHP 7.4+ features (void return types) but don't run
on production sites, so exclude them from PHP 5.4 compatibility checks.

Nikschavan committed:
fix: Revert PHPCS workflow PHP version to 7.4

Nikschavan committed:
revert: Remove version bump, keep at 1.1.9

Only updating readme metadata, no version bump needed.

Nikschavan committed:
docs: Update tested up to tag to WordPress 6.9
